### ISSUE TITLE
Inappropriate language. Use neutral & child friendly test copy.

### ISSUE THREAD

**Issue by jesserwright** [exercism/rust#919]
https://github.com/exercism/rust/blob/da24f9af56e1d52196562dedcf021f781e9d873e/exercises/bob/tests/bob.rs#L58

"Let's go make out behind the gym!" and the use of the word "HELL" I consider to be inappropriate for audiences of all ages. Let's keep it family friendly.

Please use friendly and neutral tests. Thanks :)

**Comment by coriolinus** [comment]
Good point. However, this isn't the right repo for this issue: these tests derive from the [canonical data](https://github.com/exercism/problem-specifications/blob/master/exercises/bob/canonical-data.json) for this exercise. I'm going to move this issue to that repo.
